arithmetization/src/main/java/net/consensys/linea/zktracer/types/Bytecode.java

```diff
@@ -58,7 +57,7 @@ public Bytecode(Bytes bytes) {
    */
   public Bytecode(Code code) {
     this.bytecode = code.getBytes();
-    this.hash = CompletableFuture.completedFuture(code.getCodeHash());
+    this.hash = code.getCodeHash();
```

Comment:
Could you double check with the previous version ? I think we don't initialize the hash at build time. This actually what represents the performance regression,

Reply:
It is as it was before the PR. I believe the other constructor is used mainly.

Reply:
Ah, you're right, I was referring to the other one,